server/pulp/server/managers/repo/unit_association.py

```diff
@@ -234,9 +233,7 @@ def associate_from_repo(cls, source_repo_id, dest_repo_id, criteria,
         # Now we can make sure the destination repository's importer is capable
         # of importing either the selected units or all of the units
         if not source_repo_unit_types.issubset(supported_type_ids):
-            raise exceptions.PulpCodedException(
-                error_code=error_codes.PLP0000,
-                message='The the target importer does not support the types from the source')
+            raise TypeError("The target importer does not support the types from the source")
```

Comment:
@shubham90 I propose this become a specific error_code in error_codes.py. Can you add it and bring back its usage in this module?